### DISCUSSION TITLE
Setting Signal Android Build Gradle

### DISCUSSION THREAD

**Discussion by Rahmaditha** [indrawp/Setup-Guide#86] [Q&A]
In build.gradle  we need to insert
`   buildConfigField "String", "STORAGE_URL", "\"https://cloudfrontdomain.com\""`
`   buildConfigField "String", "SIGNAL_CDN_URL", "\"https://cloudfrontdomain.com\""`

and i use minio. do i still need aws cloudfront??

        buildConfigField "String", "MRENCLAVE", "\"cd6cfc342937b23b1bdd3bbf9721aa5615ac9ff50a75c5527d441cd3276826c9\""
        buildConfigField "String", "KEY_BACKUP_ENCLAVE_NAME", "\"f2e2a5004794a6c1bac5c4949eadbc243dd02e02d1a93f10fe24584fb70815d8\""
        buildConfigField "String", "KEY_BACKUP_MRENCLAVE", "\"f51f435802ada769e67aaf5744372bb7e7d519eecf996d335eb5b46b872b5789\""

and in mrenclave what is this strings means?

**Comment by madeindra** [comment]
For `signal_cdn_url` change it to `domain.com/bucket-name`

For enclave, if you don't use CDS, ignore it
